pkg/shell/cockpit-docker.js

```diff
@@ -453,8 +453,8 @@ function render_container (client, $panel, filter_button, prefix, id, container,
     if (container.State && container.State.Running) {
         cputext = format_cpu_usage(container.CpuUsage);
 
-        memuse = container.MemoryUsage || 0;
-        memlimit = container.MemoryLimit || 0;
+        memuse = container.MemoryUsage;
+        memlimit = container.MemoryLimit;
```

Comment:
Explained: It's an unnecessary initialization of variables to default values. This makes it more difficult in the rest of the script to identify if the values have been provided by the Docker API / OS or set by Cockpit.